content/03.text-mining.md

```diff
@@ -21,34 +21,67 @@
+Within this space a hyperplane can be constructed to seperate sentences into the positive class (mentions a relationship) or negative class (does not mention a relationship). 
+Constructing these kernels necessitates the use of expert knowledge to be successful.
+Similar to regular manual curation, relying on experts to manually construct functions result in highly precise performance, but suffer overall in terms of recall.
+Besides single kernel methods, a recent study used an ensemble of SVMs to extract disease-gene associations [@doi:10.1371/journal.pone.0200699].
+This provided powerful preformance in terms of precision and outperformed the various baseline models.
+Overall, future SVM methods should consider a more automated approach to construct these kernels or develop ways to use semi-automatic methods for kenel construction.
+
+Deep learning is an increasingly popular field that can construct their own features within a high dimensional space [@raw:GoodfellowDL; @doi:10.1038/nature14539].
+These methods use different forms of neural networks, such a recurrent neural network or a convolutional neural network, to perform classifcation.
+Recurrent neural networks (RNN) are designed for sequential analysis that consist of using a repeatedly updating hidden state to make predictions.
+An example of a recurrent neural network is a long short term memory (LSTM) network [@doi:10.1162/neco.1997.9.8.1735].
+Cocos et al [@doi:10.1093/jamia/ocw180] used a LSTM to extract drug side effects from de-identified twitter posts, while Yadav et al [] used an LSTM to extract protein protein interactions.
+Other works have used LSTMs to perform relationship extraction [@arxiv:1708.03743; @doi:10.1093/bioinformatics/btw486; @arxiv:1808.09101; @doi:10.1186/s12859-017-1609-9; @doi:10.1093/jamia/ocw180]. 
+Despite the success of these networks, runtime for RNNs can be an issue.
+Using a LSTM on a dataset that has 200+ token sentences can take up to days to run.
+Based on this future approaches should be cautious on length of each individual sentence using this type of network.
+Furthermore, these methods require a lot of data to perform well.
+Future methods should consider solutions to obtaining these large quantities of data.
+
+Convolutional neural networks (CNN) are heavily used in image analysis and consist of using multiple kernel filters to capture small subsets of the overall image [@doi:10.1038/nature14539].
+In the context of text mining an image is repleaced with words within a sentence mapped to dense vectors (i.e. word embeddings) [@arxiv:1301.3781; @arxiv:1310.4546].
+Peng et al. [@arxiv:1706.01556v2] used a CNN to extract sentences concerning protein-protein intercations and Zhou et al. [@doi:10.1186/s12859-019-2873-7] used a CNN to extract chemical-disease relations.
+Other work have used this network for extraction [@doi:10.1177/0165551516673485; @doi:10.1093/database/bay073; @doi:10.1101/730085]
```

Comment:
`work` -> researchers? practitioners? something like that since it's not the work doing the reuse but people.